src/index.js

```diff
@@ -303,6 +303,5 @@ export default renderToString;
 
 export {
 	renderToString as render,
-	renderToString,
```

Comment:
This is not in document, test, type definition. 

https://github.com/preactjs/preact-render-to-string/blob/552756f827f4628489e7356a4c0b56f14d074b31/src/index.d.ts#L1-L11

https://github.com/preactjs/preact-render-to-string/blob/552756f827f4628489e7356a4c0b56f14d074b31/test/index.js#L4-L19